src/components/Pages/Login/LoginForm/index.js

```diff
@@ -104,9 +151,9 @@ export const LoginForm = () => {
             </div>
           </div>
 
-          {error && <p className={style.error__text}>{error}</p>}
+          {/* {error && <p className={style.error__text}>{error}</p>} */}
```

Comment:
Not sure if this is needed here, please check

Reply:
This was to deal with Firebase possible auth errors. I erased it for now but will add again later.